Upstream: No (it is in sourceforge)
Reason: Fixes compilation with musl libc

diff --git a/modules/urar.c b/modules/urar.c
index 206d2af..83c4660 100644
--- a/modules/urar.c
+++ b/modules/urar.c
@@ -16,6 +16,7 @@
 #include "version.h"
 
 #include <fcntl.h>
+#include <limits.h>
 #include <unistd.h>
 
 #define DOS_DIR_SEP_CHAR  '\\'

